Info form for private datasets (#213)
OpenFreeEnergy/IndustryBenchmarks2024: industry_benchmarks/utils/info_form_private_sets.txt

#################################
### Private dataset info form ###
#################################

System preparation
------------------

- Protein preparation

>   - Structure model type, e.g. X-ray, cryo-em, homology model

Author: Changed this, please let me know if this is better

  - Structure preparation, e.g. modelling of missing loops, assignment of protonation states

- Ligand preparation
  - Assignment of protonation states
  - Tautomer generation
  - Obtaining the binding mode:
        - Known binding mode: Yes/No
        - Used e.g. (constrained) docking, alignment on co-crystalized ligand, ...

> - Known binding site: [ ] Yes

Author: I put down yes and no, I would say that `no` in this case equals `unknown` or is there a case where one would want to put unknown?

Benchmark difficulty
--------------------

- Estimated benchmark difficulty
[ ] Easy
[ ] Medium
[ ] Hard

- Potential challenges
[ ] Protonation states ligands
[ ] Protonation states protein
[ ] Water sampling
[ ] Ions in the binding site
[ ] Metals in the binding site
[ ] Known sampling issues
    [ ] Ligands
    [ ] Protein
[ ] Diverse chemistry / known force field parameterization issues
[ ] Large perturbations
[ ] ...

- Additional details for the system specific challenges

Experimental assay
------------------

> - Broad assay description (e.g. “Kd from ITC”)

Author: @IAlibay : I reformatted the file, is it clearer now where things should go?

#################################
### Private dataset info form ###
#################################

System preparation
------------------

1. Protein preparation

a) Structure model type, e.g. X-ray, cryo-em, homology model:
-

b) Structure preparation details, e.g. modelling of missing loops, assignment of protonation states:
-

2. Ligand preparation

a) Method for assigning the protonation state:
-

b) Method for choosing the tautomer (if applicable):
-

c) Known binding mode: [ ] Yes  [ ] No

d) Method used for generating the binding mode (e.g. (constrained) docking, alignment on co-crystalized ligand):
-

3. Known binding site: [ ] Yes  [ ] No

Benchmark difficulty
--------------------

1. Estimated benchmark difficulty

[ ] Easy
[ ] Medium
[ ] Hard

2. Potential challenges

[ ] Protonation states ligands
[ ] Protonation states protein
[ ] Water sampling
[ ] Ions in the binding site
[ ] Metals in the binding site
[ ] Known sampling issues
    [ ] Ligands
    [ ] Protein
[ ] Diverse chemistry / known force field parameterization issues
[ ] Large perturbations

> [ ] ...

Reviewer: Maybe we could add poor ligand alignment, my thinking here is that kartograf might be changing more atoms than needed if the structures aren't very closely aligned.

Author: I added this!

#################################
### Private dataset info form ###
#################################

> 

Reviewer 2: ```suggestion

Please fill in and submit this form as part of your dataset result submission. Please make sure that non-optional sections 1c, 3, 4, 7, 8, and 9 have been completed.

```

System preparation
------------------

1. Protein preparation

> a) Structure model type, e.g. X-ray, cryo-em, homology model:

Reviewer 2: ```suggestion
a) Structure model type, e.g. X-ray, cryo-em, homology model (optional):
```

-

> b) Structure preparation details, e.g. modelling of missing loops, assignment of protonation states:

Reviewer 2: ```suggestion
b) Structure preparation details, e.g. modelling of missing loops, assignment of protonation states (optional):
```

-

2. Ligand preparation

> a) Method for assigning the protonation state:

Reviewer 2: ```suggestion
a) Method for assigning the protonation state (optional):
```

-

b) Method for choosing the tautomer (if applicable):
-

c) Known binding mode: [ ] Yes  [ ] No

> d) Method used for generating the binding mode (e.g. (constrained) docking, alignment on co-crystalized ligand):

Reviewer 2: ```suggestion
d) Method used for generating the binding mode (e.g. (constrained) docking, alignment on co-crystalized ligand) (optional):
```

-

3. Known binding site: [ ] Yes  [ ] No

Benchmark difficulty
--------------------

> 1. Estimated benchmark difficulty

Reviewer 2: ```suggestion
4. Estimated benchmark difficulty
```

[ ] Easy
[ ] Medium
[ ] Hard

> 2. Potential challenges

Reviewer 2: ```suggestion
5. Potential challenges (optional)
```

[ ] Protonation states ligands
[ ] Protonation states protein
[ ] Water sampling
[ ] Ions in the binding site
[ ] Metals in the binding site
[ ] Known sampling issues
    [ ] Ligands
    [ ] Protein
[ ] Diverse chemistry / known force field parameterization issues
[ ] Poor ligand alignment
[ ] Large perturbations

> [ ] ...

Reviewer 2: ```suggestion
[ ] <add other challenge type here>
```

> 3. Additional details for the system specific challenges:

Reviewer 2: ```suggestion
6. Additional details for the system specific challenges (optional):
```

-

Experimental assay
------------------

> 1. Broad assay description (e.g. “Kd from ITC”):

Reviewer 2: ```suggestion
7. Broad assay description (e.g. “Kd from ITC”):
```

-

> 2. Assay error provenance, e.g. multiple repeats:

Reviewer 2: ```suggestion
8. Assay error provenance, e.g. multiple repeats (if applicable):
```
Could this reveal the underlying assay type? If so we should make this optional.

-

> 3. All ligands have been measured with the same assay: [ ] Yes  [ ] No

Reviewer 2: ```suggestion
9. All ligands have been measured with the same assay: [ ] Yes  [ ] No
```

> 4. Additional details on the experimental assay:

Reviewer 2: ```suggestion
10. Additional details on the experimental assay (optional):
```

-

> Additional information (optional)
> ---------------------------------

Reviewer 2: ```suggestion
Additional information
----------------------
```

> 1. Compute hardware description, e.g. what type of GPU was used:

Reviewer 2: ```suggestion
11. Compute hardware description, e.g. what type of GPU was used (optional):
```

-

> 2. Failure information: If there were any systematic edge failures in this dataset, do you have any information about why the edge(s) failed?

Reviewer 2: ```suggestion
12. Failure information: If there were any systematic edge failures in this dataset, do you have any information about why the edge(s) failed (optional)?
```
